app/panel/actions/BlockingActions.js

```diff
@@ -18,26 +18,17 @@ import {
 	UPDATE_CATEGORY_BLOCKED,
 	UPDATE_TRACKER_BLOCKED,
 	UPDATE_TRACKER_TRUST_RESTRICT,
-	TOGGLE_EXPAND_ALL,
-	TOGGLE_EXPAND_CATEGORY
+	TOGGLE_EXPAND_ALL
```

Comment:
The SetupBlockingDropwdown component in the Hub currently uses this. It looks like we may be able to modify that component and then remove the TOGGLE_EXPAND_CATEGORY bits from the panel code, but I think I can & should do that as a separate follow-up ticket since it involves a whole other chunk of the extension.

Reply:
OK, so are you going to add the code back into BlockingActions, Constants, and Blocking.js?

Reply:
I was after all able to remove all the *TOGGLE_EXPAND_CATEGORY code. The blocking select view in the hub and the panel blocking views in settings and detailed tab all still work.